tests/sentry/api/serializers/test_sentry_app.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+
+from sentry.api.serializers import serialize
+from sentry.api.serializers.models.sentry_app import SentryAppSerializer
+from sentry.testutils import TestCase
+
+
+class SentryAppSerializerTest(TestCase):
+    def test_simple(self):
+        user = self.create_user()
+        organization = self.create_organization(owner=user)
+        sentry_app = self.create_sentry_app(
+            name="Tesla App",
+            organization=organization,
+            published=True,
+            scopes=("org:write", "team:admin"),
+        )
+        result = serialize(sentry_app, None, SentryAppSerializer(), access=None)
+
+        assert result["name"] == "Tesla App"
+        assert result["featureData"] == [
+            {
+                "description": "Tesla App can **utilize the Sentry API** to pull data or update resources in Sentry (with permissions granted, of course).",
+                "featureGate": "integrations-api",
+            }
+        ]
+        assert result["author"] == "A Company"
+        assert result["scopes"] == ["org:write", "team:admin"]
```

Comment:
@NisanthanNanthakumar Since this is the first test of this serializer, I would also make sure that you don't see the `clientSecret` since the user lacks permission for it.